bentoml/_internal/bento/docker/entrypoint.sh

```diff
@@ -17,6 +17,7 @@ _main() {
 
 	# Overide the BENTOML_PORT if PORT env var is present. Used for Heroku
 	if [[ -v PORT ]]; then
+		# NOTE: Yatai also set \$PORT, hence \$BENTOML_PORT will be overridden.
```

Comment:
Simply update the comment above: 

	# Overide the BENTOML_PORT if PORT env var is present. Used for Heroku **and Yatai**